tests/test_invenio_search.py

```diff
@@ -152,14 +166,58 @@ def test_whitelisted_aliases(app, aliases_config, expected_aliases):
     if expected_aliases == []:
         assert 0 == len(aliases)
     else:
-        assert current_search_client.indices.exists(expected_aliases)
+        for expected_alias in expected_aliases:
+            all_expected = all_aliases[expected_alias]
+            assert current_search_client.indices.exists(all_expected)
 
     app.config['SEARCH_MAPPINGS'] = orig
 
 
+@pytest.mark.parametrize(('index_name', 'suffix', 'expected'), [
+    ('authors-authors-v1.0.0', '', []),
+    ('authors-authors-v1.0.0', '-abc', []),
+    ('authors-authors-not-exist', '-test', [
+        'authors-authors-v1.0.0-test',
+        'authors-authors-v1.0.0',
+        'authors',
+    ]),
+])
+def test_creating_alias_existing_index(app, index_name, suffix, expected):
```

Comment:
It's weird how this test is passing, considering the reversed arguments used in the `.exists_alias(...)` call above... It might need some extra cases